source/monitoring.txt

```diff
@@ -12,8 +12,135 @@ Monitoring and Logging
+it, see the :ref:`<kafka-monitoring-information>` section.
+
+To view an example that shows how to monitor a running connector,
+see the :ref:`<kafka-monitoring-example>` section. 
+
+To view a list of all metrics produced by MongoDB source and sink
+connectors, see the :ref:`<kafka-monitoring-all-attributes>` section.
+
+To learn more about logging in the connector, see the
+:ref:`<kafka-logging>` guide.
+
+.. _kafka-monitoring-information:
+
+Monitor the Connector
+---------------------
+
+The {+mkc+} uses **{+jmx-long+} (JMX)** to enable monitoring.
+JMX is a technology included in the {+java-se+} that provides
+tools to monitor applications and devices. You can view the
+metrics produced by the connector with any JMX console, such
+as ``jconsole``.
+
+The {+mkc+} provides metrics for individual connector **tasks**.
+Tasks are classes instantiated by a connector that copy data to and
+from data stores and {+ak+}. A connector manages one or more tasks.
+
+To learn more about JMX, see the following resources from Oracle:
+
+- `Java Management Extensions Guide <https://docs.oracle.com/en/java/javase/17/jmx/introduction-jmx-technology.html>`__
+- `Using JConsole <https://docs.oracle.com/en/java/javase/17/management/using-jconsole.html>`__
+
+To learn more about tasks in {+kc+}, see the following resources:
+
+- `Kafka Connect Concepts from Confluent <https://docs.confluent.io/platform/current/connect/concepts.html#tasks>`__
+- `{+kc+} API documentation for Task interface <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/connector/Task.html>`__
+
+Enable Monitoring
+~~~~~~~~~~~~~~~~~
+
+The {+mkc+} uses {+kc+}'s metrics infrastructure to
+serve metrics. To read the metrics produced by your connector,
+enable JMX in your {+kc+} deployment.
+
+To learn how to enable JMX for a {+kc+} instance
+running on your host machine, see the
+`Official Kafka Documentation <https://kafka.apache.org/documentation/#monitoring>`__.
+
+To learn how to enable JMX for a containerized {+kc+}
+deployment, see
+`Kafka Monitoring and Metrics Using JMX with Docker <https://docs.confluent.io/platform/current/installation/docker/operations/monitoring.html>`__.
+
+.. _kafka-monitoring-example:
+
+Example - Monitor the Quick Start
+---------------------------------
+
+The sample environment provided in the Quick Start exposes metrics
+on your host machine at the URI ``localhost:35000``.
+
+To view these metrics with ``jconsole``, perform the following actions:
+
+.. tip:: Download jconsole
+
+   ``jconsole`` is part of the {+java-se+}. To download ``jconsole``, download the
+   `Java SE Development Kit <https://www.oracle.com/java/technologies/downloads/>`__
+   from Oracle.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Start The Quick Start Pipeline and Add Connectors
+
+      Follow the Quick Start until the :ref:`<kafka-quick-start-send-a-document>`
+      step.
+
+   .. step:: Start jconsole
```

Comment:
Issue: jconsole not in monospace here

Suggestion: Make it monospace (if Snooty lets you do that with steps)

Reply:
After reading a little further, I wonder if this should be "JConsole", since that's the name of the application you're starting (by running `jconsole`).

Reply:
You are right, this should be "JConsole" when not talking about the command. Updated throughout.